README.md

```diff
@@ -11,11 +11,18 @@ A simple seller panel to help sellers to maintain their orders
 ## Environment variables
 
 ```
+export SERVER_PORT =<Server port>
 export DB_HOST=jdbc:postgresql//<db-ip>:<db-port>/seller-panel
 export DB_USER=<Your db username>
 export DB_PASS=<Your db password>
 export REDIS_PORT=<redis service port>
 export REDIS_HOST=<redis service host>
+export MAIL_HOST =<Email service host>
+export MAIL_PORT =<Email service port>
+export MAIL_USERNAME =<Email username>
+export MAIL_PASSWORD =<Email password>
+export JWT_SECRET=<Jwt secret key>
```

Comment:
These are application specific so could be optional?

Reply:
yes other than username and password all others could be optional

Reply:
I meant that jwt secret doesn't have to be in env variables.